earth2studio/models/px/fuxi.py

```diff
@@ -440,10 +377,10 @@ def _default_generator(
             # Runs on 16 Gb V100, so seems okay
             if step == 20:
                 logger.warning(f"Time-step {step}, loading medium range model")
-                ort_session = self.create_ort_session(self.ort_medium_path, self.device)
+                ort_session = create_ort_session(self.ort_medium_path, self.device)
             elif step == 40:
                 logger.warning(f"Time-step {step}, loading long range model")
-                ort_session = self.create_ort_session(self.ort_long_path, self.device)
```

Comment:
Can we be confident that this doesn't create multiple ort sessions, that we don't need to delete a previous ort session?

Reply:
I believe the medium ORT should get garbage collected. But the short one stays in memory. I wasn't too bothered by this on a V100... but can be improved if needed.

Theres a comment above about this.

Reply:
I'll see if I can find an elegant solution to this right now

Reply:
Improved this, so there's no multiple ORT sessions repeat